benchmarks/matio_matio_fuzzer/benchmark.yaml

```diff
@@ -2,7 +2,3 @@ commit: 7a9958485036ef37b4e88df5cbe32bd7ec4c0d25
 commit_date: 2019-09-04 02:05:00+00:00
 fuzz_target: matio_fuzzer
 project: matio
-type: bug
```

Comment:
can you bring back "type: bug", that adds the sanitizer flags "-fsanitize=address,undefined".